changelogs/unreleased/6586-document-in-and-not-in.yml

```diff
@@ -0,0 +1,5 @@
+description: Document the proper way to configure a project to access a private python package repository
+issue-nr: 6586
+change-type: patch
+destination-branches: [master]
```

Comment:
The "not in" operator is new right? it's not present in previous versions I mean?